go.mod

```
@@ -0,0 +1,192 @@
+module github.com/ignite/apps
+
+go 1.21.6
```

Could we keep this in `1.21.1`?

```suggestion
go 1.21.1
```